.changeset/tasty-drinks-teach.md

```diff
@@ -0,0 +1,37 @@
+---
+'@backstage/backend-common': patch
+'@backstage/backend-tasks': patch
+'@backstage/backend-test-utils': patch
+'@backstage/cli': patch
+'@backstage/plugin-airbrake-backend': patch
+'@backstage/plugin-app-backend': patch
+'@backstage/plugin-auth-backend': patch
+'@backstage/plugin-auth-node': patch
+'@backstage/plugin-azure-devops-backend': patch
+'@backstage/plugin-badges-backend': patch
+'@backstage/plugin-bazaar-backend': patch
+'@backstage/plugin-catalog-backend': patch
+'@backstage/plugin-catalog-backend-module-aws': patch
+'@backstage/plugin-catalog-backend-module-azure': patch
+'@backstage/plugin-catalog-backend-module-bitbucket': patch
+'@backstage/plugin-catalog-backend-module-github': patch
+'@backstage/plugin-catalog-backend-module-gitlab': patch
+'@backstage/plugin-code-coverage-backend': patch
+'@backstage/plugin-graphql-backend': patch
+'@backstage/plugin-kafka-backend': patch
+'@backstage/plugin-kubernetes-backend': patch
+'@backstage/plugin-periskop-backend': patch
+'@backstage/plugin-permission-backend': patch
+'@backstage/plugin-permission-node': patch
+'@backstage/plugin-proxy-backend': patch
+'@backstage/plugin-rollbar-backend': patch
+'@backstage/plugin-search-backend-module-pg': patch
+'@backstage/plugin-search-backend-node': patch
+'@backstage/plugin-tech-insights-backend': patch
+'@backstage/plugin-techdocs-backend': patch
+'@backstage/plugin-techdocs-node': patch
+'@backstage/plugin-todo-backend': patch
+---
+
+Update to `jose` 4.6.0
+Require jest tests to use node (rather than jsdom)
```

Comment:
The Jest `node` environment is being fixed as part of https://github.com/backstage/backstage/pull/9255 in a slightly different way. We'll likely merge that PR as soon as it's ready to go. In the meantime you could keep the config here around in this PR and remove it once the Jest 27 bump is in master.

Reply:
Alright it's now merged, so all node packages should now be running with a node environment if you rebase on latest master

Reply:
That's great, I rebased and discarded all the unneeded changes.